Code/Framework/AzCore/AzCore/DOM/DomPrefixTree.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/DOM/DomPath.h>
+#include <AzCore/std/containers/map.h>
+#include <AzCore/std/containers/stack.h>
+#include <AzCore/std/optional.h>
+
+namespace AZ::Dom
+{
+    //! Specifies how a patch matches against a DomPrefixTree
```

Comment:
Does this mean that the patch approach in the GenericDOM is going to be a superset of JSON Patch?

Reply:
That was an unfortunate typing mistake! This is just referring to path matches - while Generic DOM patches may eventually become supersets of JSON patch, this work is unrelated.